src/Tribe/Main.php

```diff
@@ -575,11 +573,17 @@ public function hooks() {
 			add_filter( 'tribe_event_import_rsvp_column_names', [ Tribe__Tickets__CSV_Importer__Column_Names::instance(), 'filter_rsvp_column_names' ] );
 		}
 
-		// Load our assets
-		add_action( 'tribe_tickets_plugin_loaded', tribe_callback( 'tickets.assets', 'enqueue_scripts' ) );
-		add_action( 'tribe_tickets_plugin_loaded', tribe_callback( 'tickets.assets', 'admin_enqueue_scripts' ) );
-		add_action( 'admin_enqueue_scripts', tribe_callback( 'tickets.assets', 'enqueue_editor_scripts' ) );
-		add_filter( 'tribe_asset_data_add_object_tribe_l10n_datatables', tribe_callback( 'tickets.assets', 'add_data_strings' ) );
+		/**
```

Comment:
Did we need this change?

Reply:
_need?_ no. helps IDE though.

Reply:
IDE maybe but the difference between tribe() and tribe_callback() is one sets up the object, one is a callback that sets up the object and calls the method. I could be wrong and maybe the assets object gets setup ahead of time so this is negligible but the replacement of tribe_callback() is something we'll want to weigh carefully.

Reply:
@sc0ttkclark 
Isn't that what `$assets = tribe( 'tickets.assets' );` is doing (just below this line in GitHub's inline commenting)?

**Are you saying these 2 are slightly different?**

Before:
![Screen Shot 2019-12-08 at 12 31 57 PM](https://user-images.githubusercontent.com/1812179/70394074-df573c80-19b6-11ea-8129-3afcfb0b44e7.png)


After:
![image](https://user-images.githubusercontent.com/1812179/70394049-9ef7be80-19b6-11ea-9c05-f677954ac44f.png)

Reply:
Yes, tribe_callback() returns a temporary callback reference, it does not setup tribe() for that associated container until the callback function is ultimately called via PHP when the action itself runs.

Reply:
tyvm for clarifying